arenadata/Dockerfile.ubuntu

```diff
@@ -29,27 +29,42 @@ RUN set -eux; \
     pip2 install allure-behave==2.4.0; \
     pip2 cache purge
 
+RUN wget https://www.python.org/ftp/python/3.9.19/Python-3.9.19.tgz -O - | tar zxf - && \
+    cd Python-3.9.19 && \
+    ./configure --prefix=/usr/local --enable-shared --enable-optimizations && \
+    make -j6 && \
+    make install && \    
+    echo "/usr/local/lib" >>/etc/ld.so.conf.d/gpdb.conf && \
+	ldconfig && \
+    cd .. && \
+    rm -rf Python-3.9.19.tgz Python-3.9.19
```

Comment:
```suggestion
    rm -rf ../Python-3.9.19.tgz ../Python-3.9.19
```

Reply:
Can not fully agree, you suggest to remove current directory, it will work, but looks not logical for me. And some OSes (not relevant for this project although) does not allow it